inference-engine/src/mkldnn_plugin/nodes/mkldnn_pooling_node.cpp

```diff
@@ -273,11 +273,8 @@ void MKLDNNPoolingNode::initSupportedPrimitiveDescriptors() {
                 dataConfig.inPlace = -1;
                 dataConfig.constant = false;
                 auto desc = getSrcMemDesc(itpd, i);
-                if (desc->getType() & MemoryDescType::Blocked) {
-                    dataConfig.desc = desc->as<BlockedMemoryDesc>()->cloneWithUndefStridesAndOffset();
-                } else {
-                    dataConfig.desc = std::move(desc);
-                }
+                dataConfig.desc = std::move(desc);
```

Comment:
why did you remove undefined descs creation for pooling node? I guess we need to restore it.